src/controllers/connectionManager.ts

```diff
@@ -523,7 +523,7 @@ export default class ConnectionManager {
                     self.connect(fileUri, connectionCreds)
                     .then(result => {
                         self.handleConnectionResult(result, fileUri, connectionCreds).then(() => {
-                            resolve(true);
+                            resolve(connectionCreds);
                         });
```

Comment:
You never seem to actually use the returned creds so I'd just remove this whole .then() since it's not doing anything